Description: Downgrade maven-scm-publish-plugin to build with Maven 2
Last-Update: 2014-07-29
--- a/pom.xml
+++ b/pom.xml
@@ -72,7 +72,7 @@
         <coveralls-maven-plugin.version>2.2.0</coveralls-maven-plugin.version>
         <maven-release-plugin.version>2.5</maven-release-plugin.version>
         <netty-all.version>4.0.12.Final</netty-all.version>
-        <maven-scm-publish-plugin.version>1.1</maven-scm-publish-plugin.version>
+        <maven-scm-publish-plugin.version>1.0-beta-2</maven-scm-publish-plugin.version>
         <tempus-fugit.version>1.1</tempus-fugit.version>
         <maven-dependency-plugin.version>2.8</maven-dependency-plugin.version>
         <maven-assembly-plugin.version>2.2-beta-5</maven-assembly-plugin.version>
@@ -323,4 +323,4 @@
             </plugin>
         </plugins>
     </reporting>
-</project>
\ No newline at end of file
+</project>
